packages/flutter/lib/src/rendering/object.dart

```diff
@@ -507,7 +510,10 @@ class PaintingContext extends ClipContext {
     assert(clipBehavior != null);
     final Rect offsetBounds = bounds.shift(offset);
     final Path offsetClipPath = clipPath.shift(offset);
```

Comment:
This work is wasted if behaivor is none. Should we skip it in that case?

Reply:
(same above)

Reply:
done